src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -99,14 +99,19 @@ public interface DatasetDAO extends Transactional<DatasetDAO> {
     @SqlUpdate("DELETE FROM dataset WHERE datasetid = :datasetId")
     void deleteDatasetById(@Bind("datasetId") Integer datasetId);
 
-    @SqlUpdate("update dataset set active = :active where dataSetId = :dataSetId")
-    void updateDatasetActive(@Bind("dataSetId") Integer dataSetId, @Bind("active") Boolean active);
-
-    @SqlUpdate("update dataset set needs_approval = :needs_approval where dataSetId = :dataSetId")
-    void updateDatasetNeedsApproval(@Bind("dataSetId") Integer dataSetId, @Bind("needs_approval") Boolean needs_approval);
-
-    @SqlUpdate("UPDATE dataset SET update_date = :updateDate, update_user_id = :updateUserId WHERE datasetid = :datasetId")
-    void updateDatasetUpdateUserAndDate(@Bind("datasetId") Integer datasetId, @Bind("updateDate") Timestamp updateDate, @Bind("updateUserId") Integer updateUserId);
+    @SqlUpdate("UPDATE dataset SET active = :active WHERE dataSetId = :datasetId")
```

Comment:
Optional: here and in other queries, `dataSetId` can be all lower cased. JDBI/postgres will do that for you when the actual query is executed, so this work isn't necessarily required - it's just better form.
```suggestion
    @SqlUpdate("UPDATE dataset SET active = :active WHERE datasetid = :datasetId")
```